avbroot/src/patch/boot.rs

```diff
@@ -510,7 +510,7 @@ impl BootImagePatch for MagiskRootPatcher {
         magisk_config.push_str("SHA1=0000000000000000000000000000000000000000\n");
 
         if Self::VER_RANDOM_SEED.contains(&self.version) {
-            magisk_config.push_str(&format!("RANDOMSEED={:#x}\n", self.random_seed));
+            write!(&mut magisk_config, "RANDOMSEED={:#x}\n", self.random_seed).unwrap();
```

Comment:
Using `unwrap()` here would result in `avbroot` to panic if `random_seed` is `null` or `undefined`. Isn't it?

Reply:
In this case, the `unwrap()` is for the result of `write!()`, which cannot fail when appending to a string like `magisk_config`. (It can fail if writing to some other target, like a file.)

Regarding `random_seed`, there's no concept of null or undefined in Rust. It is always a valid integer.